build/Analyzers.ruleset

```diff
@@ -2,57 +2,31 @@
 <RuleSet Name="Autofac Analyzer Rules" Description="Analyzer rules for Autofac assemblies." ToolsVersion="16.0">
   <IncludeAll Action="Warning" />
   <Rules AnalyzerId="Microsoft.Usage" RuleNamespace="Microsoft.Usage">
-    <!-- Implement standard exception constructors - not all of the exception constructors (e.g., parameterless) are desired in our system. -->
```

Comment:
This seems to happen a lot - the comments explaining what the rules are and why we've omitted them get dropped. I find it helpful to know what the rules are; unclear if others do. Might need to stop editing this one in VS.

Reply:
I somehow manage to do this every single time, I need to put a post-it on my monitor not to open the rulesets file in VS...